integr8ly/installation
# INTLY-3248 - Remove SSO link from OCP web console

## Additional Information
JIRA: https://issues.jboss.org/browse/INTLY-3248

## Verification Steps

- Get Integreatly cluster from PDS
- Open Openshift web console and verify that "Red Hat Single Sign-On" is in the list of apps on top left and in the drop-down on top right. Screenshot:

 
![Screenshot 2019-09-12 at 11 17 01](https://user-images.githubusercontent.com/1605799/64771305-0684a600-d54f-11e9-89d9-04e193d3e9e9.png)

- SSH to bastion node, switch to root user, pull this PR and execute `cd installation`
- Update `[master]` part of the `inventories/hosts.template` file (you will need to put local address of your master, e.g. `master1.omatskiv-8f86.internal`) 
- Execute `ansible-playbook -i ./inventories/hosts.template ./playbooks/customise_web_console_install.yml`
- Open Openshift web console again and refresh the page
- Verify that "Red Hat Single Sign-On" links are no longer present
- Verify that other application links work

## Is an upgrade task required and are there additional steps needed to test this?

Upgrade changes are included. Verification steps will verify both the upgrade path, and clean installation.
